src/test/java/tests/TestUtils.java

```diff
@@ -77,4 +83,51 @@ public static void delay(double seconds) {
         }
     }
 
+    public static String readFileFromResource(Object callingObject, String filename) throws IOException {
+        ClassLoader classLoader = callingObject.getClass().getClassLoader();
+        return IOUtils.toString(classLoader.getResourceAsStream(filename));
+    }
+
+    /**
+     * Parses a string containing HTTP header fields and return a List of Header to be used with MockServer
+     * @param header
+     * @return
+     */
+    public static List<Header> parseHeaderRecord(String header) {
+        List<Header> result = new ArrayList<>();
+        String[] fields = header.split("[\r\n]+");
+
+        for (String field : fields) {
+            String[] nameAndValue = field.split(":", 2);
+            if (nameAndValue.length == 2) {
+                result.add(new Header(nameAndValue[0].trim(), nameAndValue[1].trim()));
+            }
+        }
+
+        return result;
+    }
+
+    /**
+     * Creates a GitHub API v3 request that can be used with MockServer and GitHubClient
+     * @param method the HTTP request method
+     * @param page the page of the resource being requested
+     * @param repoStringId string version of the repo id
+     * @param repoNumericId numeric version of the repo id returned in the Link header
+     * @param apiSegments segments of the request path after the repo id
+     * @return
+     */
+    public static HttpRequest createMockServerRequest(String method, String page,
```

Comment:
GitHub API doesn't guarantee that pages are labeled with numeric values although we can see that they are. It's the same case with numeric id, we can observe from the urls that they look like a number but it's not documented that they must be or how large the number can be.

I think we can change the page to `int`, but the numeric id may better be left as `String`.